data/sql/updates/pending_db_world/Horrified_Drakkari.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE `creature_template` SET `faction` = 37, `flags_extra` = 66 WHERE (`entry` = 26582);
+UPDATE `creature_template` SET `faction` = 37, `flags_extra` = 66 WHERE (`entry` = 26583);
```

Comment:
Why are you setting the `faction` here? You're not changing it, you don't need to set it.

Also, since you're changing the `flags_extra` for these creatures from 64 to 66 (adding 2), [as it is a bitmask](https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits), you should instead do:
```
Old: `flags_extra` = 66
New: `flags_extra` = `flags_extra`|2
```

Reply:
> Do i need to change the SQL?

I would recommend it, yes.

Reply:
I apologize for being unclear, I meant more the following:
```sql
UPDATE `creature_template` SET `flags_extra` = `flags_extra`|2 WHERE `entry` IN (26582, 26583);
```